### PR TITLE
Config and logging cleanups

### LINKED ISSUE #119: Maybe use the log file as stdout for the daemon
So it catches panics without our hacks to log on panics..

### PR DESCRIPTION
**Pull request by darosior**
Some cleanups that i cherry-picked from my "Spend process" branch as the drive-by fixups started to accumulate :)

Fixes #119

### CODE REVIEW

**Comment by darosior** [comment]
Added 2 drive-by fixes for the CI, i think cf3b3d1 can help with #118

File: src/daemon/bitcoind/actions.rs [hunk 1]
```diff
@@ -291,11 +291,10 @@ fn comprehensive_rescan(
                     MIN_CONF,
                 );
                 db_unconfirm_deposit_dbtx(db_tx, vault.id)?;
-                assume_some!(
-                    deposits_cache.get_mut(&vault.deposit_outpoint),
-                    "Db vault not in cache?"
-                )
-                .status = VaultStatus::Unconfirmed;
+                deposits_cache
```
Comment (danielabrozzoni) [inline src/daemon/bitcoind/actions.rs:294]: :%s/assume_ok/assume_some in the commit message

Comment (darosior) [reply src/daemon/bitcoind/actions.rs:294]: Yeah... I meant to clean both but you are right i only pruned `_some`s! Done.

Comment (darosior) [reply src/daemon/bitcoind/actions.rs:294]: Hope CI roulette not gonna bite me now :sob:

**Comment by danielabrozzoni** [comment]
ACK 4b80ba0100da00c4c55a26dbdc04157fdc88e21c

**Review by danielabrozzoni — APPROVED** [review]

**Pull request merged**